### ISSUE TITLE
Is your project  for mobile platform?

### ISSUE THREAD

**Issue by issxjl2015** [Nielsyang/YOLOv1_tensorflow#2]
in you YOLOv1_tensorflow project, that is your all file? if I downloads, can run in directly?thank you!

**Comment by lesliejackson** [comment]
No, it doesn't support mobile plat. Actually, it's just a toy implementation, ugly and slow. if you want to do object detection on mobile plat, maybe [tensorflow android](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/examples/android) and [tensorflow object detection](https://github.com/tensorflow/models/tree/master/object_detection) will help you
